src/Components/AdvancedSearch/Internal/ColumnPicker/ColumnPicker.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+    IColumnPickerProps,
+    IColumnPickerStyleProps,
+    IColumnPickerStyles
+} from './ColumnPicker.types';
+import { getStyles } from './ColumnPicker.styles';
+import {
+    classNamesFunction,
+    useTheme,
+    styled,
+    Icon,
+    Dropdown,
```

Comment:
when no columns are selected, we should still get a message. Maybe something like. "Select additional columns"
![image](https://user-images.githubusercontent.com/57726991/193125879-af7dc38f-5fa7-46d0-b4e3-5f3660f12636.png)